botocore/response.py

```diff
@@ -110,6 +110,22 @@ def read(self, amt=None):
             self._verify_content_length()
         return chunk
 
+    def readinto(self, b):
```

Comment:
Hi @aJanuary,

Sorry for doing this over two rounds of review a few weeks apart; this PR addressed all of the subclasses when you first made it, but a few months ago we added a rolling checksum to s3 streams in the `StreamingChecksumBody` class.  You can see the code for the read method [here](https://github.com/boto/botocore/blob/df53fce76a2e7b6e4fbc5d89024880761352d6fd/botocore/httpchecksum.py#L239-L244).

If we were to merge this today, anyone using `readinto` would not be using checksums by default.  Would you be open to adding an implementation and some tests in the subclass as well?  Or potentially taking a PR to your fork that will add support for this?

Reply:
Looks good to me, just finishing up some manual testing now.  Unless I find any issues with this, we should be able to get it out in a release early next week.  

Thank you for your responsiveness here and your contribution